src/components/Views.jsx

```diff
@@ -40,8 +46,29 @@ const Views = ({
             />
         </Menu.Title>
         <Menu.Body>
+            <Dialog className="cq-dialog cq-view-dialog">
```

Comment:
Placing this inside `<Menu.Body>` resizes the menu dropdown when the prompt shows up. The dropdown and the dialog needs to be side by side.